nautobot/users/filters.py

```diff
@@ -69,6 +69,28 @@ def search(self, queryset, name, value):
         )
 
 
+class TokenFilterSet(BaseFilterSet):
+    q = django_filters.CharFilter(
+        method="search",
+        label="Search",
+    )
+    user_id = django_filters.ModelMultipleChoiceFilter(
+        field_name="user",
+        queryset=get_user_model().objects.all(),
+        label="User",
+    )
+    user = django_filters.ModelMultipleChoiceFilter(
```

Comment:
As I was reviewing the test criteria, since we are removing the ability for superusers to see all user API tokens, I don't know if this will do anything anymore. @glennmatthews Thoughts on leaving this is for potential future uses?

The `TokenViewSet` as implemented will already do user filtering for non-superusers.

Reply:
I think you're right - we could probably:

- Remove `user_id` and `user` from TokenFilterSet
- Remove `user` from `TokenSerializer`
-  Update the example in `authentication.md` accordingly.